package.json

```diff
@@ -15,5 +16,9 @@
     "gulp": "^4.0.2",
     "gulp-cli": "^2.3.0",
     "gulp-concat": "^2.6.1"
+  },
+  "devDependencies": {
+    "@playwright/test": "^1.47.2",
```

Comment:
I think most of this commit can go? We don't need JS Playwright at all, do we?

Reply:
Removed with https://github.com/DemocracyClub/electionleaflets/pull/257/commits/55f6a1f8661a879b1512904d0c8c294bf7043dc0